tests/runs/load.txtar

```diff
@@ -0,0 +1,24 @@
+["cat", "dog", "cow", "pig", "sheep", "chicken", "duck", "horse", "human"]
+["meow", "woof", "moo", "oink", "baa", "cluck", "quack", "neigh", "oi oi oi"]
```

Comment:
:)

I sometime use [animal group names](https://arapahoelibraries.org/blogs/post/names-for-groups-of-animals/)

Reply:
no humans

Reply:
"Cats: a clowder, glaring, pounce, nuisance or clutter; Kittens: a litter or kindle; Wild cats: a destruction"

a destruction hahaha